fractal_tasks_core/copy_ome_zarr.py

```diff
@@ -91,6 +91,7 @@ def copy_ome_zarr(
         raise NotImplementedError
 
     if ROI_table_names is None:
+        # FIXME: are these defaults OK?
```

Comment:
Can we briefly discuss when we hit this case? Don't fully follow when we would have no ROI_table_names.

Also, does that mean only does ROI tables are copied?
(if we don't do MIPs for labels, then it can make sense to not have other ROI tables I guess)

Reply:
Agreed - and related to a different task